docs/source/algorithms/Bin2DPowderDiffraction-v1.rst

```diff
@@ -0,0 +1,128 @@
+
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm performs binning of the given workspace in :math:`(d, d_{\perp})` space, where :math:`d` is the d-Spacing and
+
+.. math::
+   :label:
+
+    d_{\perp} = \sqrt{\lambda^2 - 2\log\cos\theta}
+
+The result is that a Matrix workspace is created. If :math:`\theta=0` or :math:`\cos\theta\le 0`, the algorithm will terminate with an
+error message, since no reasonable :math:`d` or :math:`d_{\perp}` value can be calculated. In this case it is recommended to check
+whether detector positions are correct and to mask the problem detectors.
+
+
+More details on the multidimensional TOF powder diffraction data reduction can be found in:
+
+[1]  *J. Appl. Cryst.* (2015) **48**, pp. 1627-1636
+
+[2]  *J. Appl. Cryst.* (2017) **50**, pp. 866-875.
```

Comment:
Format these similar to [SetSampleMaterial-v1.rst](https://github.com/mantidproject/mantid/blob/master/docs/source/algorithms/SetSampleMaterial-v1.rst)